src/data/endpoints/managed.yaml

```diff
@@ -0,0 +1,83 @@
+# vim: set et ts=2 sw=2 tw=80 :
+name: Managed
+sort: 1
+description: >
+  Managed service endpoints provide a means of manipulating the information related to your Managed service preferences.
+endpoints:
+  /managed/contacts:
+    group: Contacts
+    type: resource
+    description: >
+      Manage your managed service contacts list.
+    methods:
+      GET:
+        response: ManagedContact
+        description: >
+          Returns your contact list.
+        examples:
+          curl: |
+            curl https://$api_root/$version/managed/contacts
+      POST:
+        description: >
+          Registers a new contact to your managed service contacts list.
+        params:
+          name:
+            description: The contact's full name.
+            type: String
+            limit: "1-64 characters"
+          email:
+            description: The email address for the contact.
+            type: String
+          phone1:
```

Comment:
should be "primary" and in a "phone" envelope

Reply:
this should actaully be `phone_primray`